docs/sphinx_api_docs_source/utils.py

```diff
@@ -61,6 +61,18 @@ def apply_structure_changes(soup, html_file_path, html_file_contents):
         parent_div = soup.select("#properties")[0]
         parent_div.append(table)
 
+    # Display signatures as code blocks
+    items = soup.select(".sig-object")
+    for item in items:
+        code_block = soup.new_tag("CodeBlock", language="python", title="Signature")
+        code_block.append("{`" + item.get_text().replace("#", "") + "`}")
+        item.replace_with(code_block)
+
+    # Prevent build from failing when there's code-like text outside code blocks
+    for item in soup.find_all(text=True):
+        if item.string and ("CodeBlock" not in item.string):
+            item.string.replaceWith(item.get_text().replace("<", r"\<"))
```

Comment:
Why is this happening?

Reply:
There are some pages that have code-like syntax in the text (for example https://docs.greatexpectations.io/docs/reference/api/data_context/data_context/context_factory) in which we have a code usage example:
```
import great_expectations as gx

my_context = gx.get_context(<insert_your_parameters>)
```
The use of JSX in the markdown misleads Docusaurus to think that it should interpret `<insert_your_parameters>` as a component and tries to render it, resulting in syntax error as it's no valid code and does not have a closing tag